src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -240,11 +240,12 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             // relevant ports.
             var connectors = obj["Connectors"].ToObject<IEnumerable<ConnectorModel>>(serializer);
 
-            // annotations
-            var annotations = obj["Annotations"].ToObject<IEnumerable<AnnotationModel>>(serializer);
-
             var info = new WorkspaceInfo(guid.ToString(), name, description, Dynamo.Models.RunType.Automatic);
 
+            //Build an empty annotations. Annotations are defined in the view block. If the file has View block
+            //serialize view block first and build the annotations.
+            var annotations = new List<AnnotationModel>();
```

Comment:
WorkspaceModel requires annotations. I believe after the View block serialization, annotationModel  will be added to workspaceModel, and the viewModel will get constructed.